Cargo.toml

```diff
@@ -159,6 +159,7 @@ base64 = "0.22.1"
 tailcall-hasher = { path = "tailcall-hasher" }
 serde_json_borrow = "0.3.0"
 Inflector = "0.11.4"
+indenter = "0.3.3"
```

Comment:
Do we need this? It seems like it's only used to implement Debug.

Reply:
yes, this only for debug, but debug is valuable